From: Evgeni Golov <evgeni@debian.org>
Description: Search for plugins in /usr/lib/geany
 Since debhelper compat level 9, plugins are installed and searched in
 /usr/lib/<arch-triplet>/geany.
 Current geany-plugins packaging, and more important, 3rd party stuff
 is still using /usr/lib/geany, thus search there too.
Forwarded: not-needed
Last-Update: 2011-11-05

Index: geany/src/plugins.c
===================================================================
--- geany.orig/src/plugins.c	2021-10-18 11:24:09.936803988 +0800
+++ geany/src/plugins.c	2021-10-18 11:24:09.936803988 +0800
@@ -1216,6 +1216,7 @@
 
 	/* finally load plugins from $prefix/lib/geany */
 	load_plugins_from_path(plugin_path_system);
+	load_plugins_from_path("/usr/lib/geany");
 
 	/* It is important to sort any plugins that are proxied after their proxy because
 	 * pm_populate() needs the proxy to be loaded and active (if selected by user) in order
